README.md

```diff
@@ -45,6 +45,8 @@ and Erlang/OTP.
 | ubuntu-16.04 | 17 - 24    | ✅
 | ubuntu-18.04 | 17 - 24    | ✅
 | ubuntu-20.04 | 20 - 24    | ✅
+| windows-2016 | 23 - 24    | ✅
```

Comment:
Yeah 😢 
I mean, we could try and fetch 22, for example, but then it's not saved in the same "C:\Program Files\" folder, so I prefer to do this initially, to have no exception, and ask for your opinion on this choice

Reply:
23+ sounds fine to me. I'd even make it 24+ if that'd simplify anything. As a matter of fact, I'd make it windows-2019 & 24+ if that'd simplify anything. We literally didn't have any other options for Windows so this more than what we could hope for anyway.

Reply:
> I'd even make it 24+ if that'd simplify anything.

It wouldn't 😄. I started with 19, actually, and most of the process works (download, install, etc.), but then, since the path is not easy to figure out - I don't know the equivalent of `find` in PowerShell and was lazy to try it out - I started bumping it until I came to this.

Reply:
cool, lets leave things as is.